studio-docs/source/org/members.mdx

```diff
@@ -7,13 +7,9 @@ A Studio organization can have any number of members, and each member can be ass
 
 ## Organization-wide member roles
 
-> **Member roles are available for organizations with an Enterprise plan.**
->
-> Apollo Support can also grant complimentary access to the Billing Manager role for Team accounts on request. Please contact **support@apollographql.com** for more information.
+> **Billing Mananger and Org Admins are available to all organizations, the other roles are available for organizations with an Enterprise plan.**
```

Comment:
typo `Mananger` –– maybe you meant `Banananger`? jk :P

We might want to take the other approach here and say our `Consumer` and `Org Admin` roles are available to all accounts, but the more granular roles of `Observer`, `Contributor` and `Graph Admin` are only available ot organizations with an Enterprise plan.

I don't think we need to call out `Billing Manager` as explicitly, that one will be reasonably assumed to be free.

Reply:
oops! Banananger :D

I merged the other PR into this one, sooo they can go out together and updated the text to closer to what you said! 👍